_release-pypi/action.yml

```diff
@@ -22,6 +22,10 @@ inputs:
     description: 'Desired Python version for using twine'
     required: true
     type: string
+  use-cache:
+    description: 'Whether to use Python chache or not. Requires a pyproject.toml file.'
+    required: false
+    type: boolean
```

Comment:
You may need to add the default option.

Reply:
I would default it to false probably. But that's just my point of view 😄

Reply:
I just imposed required parameters for "private" actions.

Actions using this one have a `use-python-cache = true` by default. This way we avoid passing one additional parameter and try to speed up our CI.